scene/main/scene_tree.cpp

```diff
@@ -1837,7 +1841,7 @@ void SceneTree::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("get_node_count"), &SceneTree::get_node_count);
 	ClassDB::bind_method(D_METHOD("get_frame"), &SceneTree::get_frame);
-	ClassDB::bind_method(D_METHOD("quit"), &SceneTree::quit);
+	ClassDB::bind_method(D_METHOD("quit", "exit_code"), &SceneTree::quit, DEFVAL(-1));
```

Comment:
Makes sense to me, but I wonder if it's then still relevant to have `set_exit_code()` exposed in `OS`. Is there any situation where one would set the error code manually prior to calling `quit()`, if `quit()` can take the error code itself?

Reply:
I don't think so, but removing it will break compatibility with existing projects.

Reply:
Ah right, I thought it had been added during 3.2 development, but it's older.